content/documentation/concensus-update-process.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Consensus Update Process
+description: Consensus Update Process | Iron Fish Blockchain
+---
+
+To maintain consensus, all full nodes validate blocks using the same consensus rules. However, sometimes the consensus rules are changed to introduce new features or prevent network abuse. A hard fork will be performed when the new consensus rules are introduced. Hard fork creates
+a permanent divergence in the block chain, where non-upgraded nodes can’t validate blocks created by upgraded nodes that follow newer consensus rules.
+
+### Requirement for Hard fork
+1. Hard Fork FIP, including the full list of changes in the consensus
+2. New randomness for the circuit if trust setup is required for the hard fork
```

Comment:
This is not related to hard forks, this is related to circuit changes.

Reply:
I will rephrase it to `New randomness for the circuit if hard fork has any circuit change`, what do you think?